src/contracts/AccessControlLP.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {MultiLP} from "./MultiLP.sol";
+
+abstract contract AccessControlLP is MultiLP {
```

Comment:
As mentioned earlier, we will plan on moving this out to a separate contract.

Reply:
And making it only affect deposits/mints.

Reply:
yep, I'll move this out today